thirdparty/google-styleguide/run_linter.py

```diff
@@ -0,0 +1,27 @@
+import argparse
+import os
+import cpplint
+
+script_dir = os.path.dirname(os.path.realpath(__file__))
+parser = argparse.ArgumentParser()
+
+parser.add_argument('-i', '--includePathes', nargs="+", help='Run linter check in these directories')
```

Comment:
typo: includePathes -> includePaths

Reply:
changed it in [e2a4e48](https://github.com/martinzink/nifi-minifi-cpp/commit/e2a4e487ce85305894d3ba7fc668f5d4f73a9821)